### ISSUE TITLE
Undefined is not an object 

### ISSUE THREAD

**Issue by katyaa** [NeXTs/Clusterize.js#128]
I am getting the following error TypeError: undefined is not an object (evaluating 'this.content_elem.lastChild.style.height = data.bottom_offset + 'px'')

Stack trace:
InsertToDOM
scrollEv

I have a list of 7196 elements. If I slice the list to only 200 elements, Clusterize works fine. But with the full size, I get the error above.

**Comment by NeXTs** [comment]
Hey
Can I see demo at codepen please?

**Comment by NeXTs** [comment]
how do you slice?

https://codepen.io/NeXTs/pen/jaYMMK

**Comment by katyaa** [comment]
Hi Denis, here's the short demo https://codepen.io/kempkens/pen/jaYwoY
It's pretty much mimicking how we use it in our codebase. I don't slice, I was doing it just to see if the problem is related to the size of the list. Thanks!

**Comment by NeXTs** [comment]
https://codepen.io/NeXTs/pen/mqpMxw

Those classnames are quite important
Also I've included clusterize's styles

Are you sure problem was in that?

**Comment by katyaa** [comment]
I just copied your pen over and it still wasn't working for me until I copied the css resource as well (https://cdnjs.cloudflare.com/ajax/libs/clusterize.js/0.18.0/clusterize.css). With that included it works. 

I'm using npm to include clusterize dependency in my package.json:
"devDependencies": {
    ...
    "clusterize.js": "0.18.0",
   ...
}

Anything else I should be adding to include the css as well? Thanks so much!

**Comment by katyaa** [comment]
Hi Denis, I added the stylesheet, but it didn't help solve the problem. I ended up copying the js files and debugging it. In the end it turned out there was an extra white space after the closing li tag, which caused the lastChild element to be a textObject and not the list object, which resulted in the style property being undefined. I fixed my template and now everything works perfectly. Thanks for your help!
